frontend/src/modules/Metadata_Forms/components/MetadataFormFields.js

```diff
@@ -661,11 +721,16 @@ export const MetadataFormFields = (props) => {
               <Autocomplete
                 disablePortal
                 options={versionOptions.map((option) => {
-                  return { label: 'version ' + option, value: option };
+                  return {
+                    label: 'version ' + option.version,
+                    value: option.version,
+                    attached: option.attached_forms
+                  };
```

Comment:
can we wrap this piece in a loading block similar to how we do elsewhere - otherwise when a user first clicks on Fields Tab it says `Version 0` and then re-updates to the latest version (a bit confusing UX)

Or we can just set default version to be null and handle the case when currentVersion